src/backend/app.py

```diff
@@ -45,15 +49,17 @@ def register():
         result = user_login.register(username, email, password, phone)
         if not result.status:
             return result.message, 400
+        session["username"] = username
         return result.message, 201
     return "", 400

```

Comment:
why are you making a session here

Reply:
Nvm removed that now, figured out how to delay register long enough to tell users that they will go to login.